### PR TITLE
adding grid component

### PR DESCRIPTION
**Pull request by bipin7yadav**
Grid

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
good work👍

File: grid.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+
+
+body{
+    position: relative;
+    margin-left: .2rem;
+    width: 100%;
+}
+
+.body{
```
Comment (MarkVed17) [inline grid.css:9]: this works, but you should've thought about a relevant class name here.

File: grid.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel="stylesheet" href="grid.css">
+    <title>Document</title>
```
Comment (MarkVed17) [inline grid.html:10]: Giving titles to pages is a good practice. Try giving some names you could think of.

**Review by MarkVed17 — APPROVED** [review]
This works! The code isn't well formatted in some places. A formatted code makes reading it simpler. You're good to proceed.

File: grid.html (hunk 2 above)
Comment (bipin7yadav) [reply grid.html:10]: sure

**Pull request merged**